substrate/client/merkle-mountain-range/src/test_utils.rs

```diff
@@ -154,7 +154,7 @@ impl MockClient {
 					parent_hash,
 				);
 				offchain_db.local_storage_set(
-					StorageKind::PERSISTENT,
+					RuntimeInterfaceStorageKind::PERSISTENT,
```

Comment:
Sorry but the storage kind + Http error should NOT be renamed. This is just wrong. Because these types are not just used for the runtime interface, but also in the rest of the code base.

Reply:
No worries, renamed them back!